src/system_error.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#define __STDC_WANT_LIB_EXT1__ 1
```

Comment:
What is this used (or required) for?

Reply:
libc++ uses `strerror_s`, but that's only provided if that is defined due to it being an extension.

Reply:
But why is this specific to nxdk? Shouldn't upstream libcxx also require this?

Reply:
No, because it's only used for msvc, and that provides it without having to request the extension.

Reply:
I'm still not happy about this line and I still think upstream should handle this.
Therefore I'd have preferred if this was a separate commit or had some sort of comment.

*But it's a minor issue and me being pedantic. So I'm fine with keeping it the way it is.*